src/app/shared/services/auth.service.ts

```diff
@@ -80,12 +80,12 @@ export class AuthService extends BaseBackendService<BaseModel> {
   public logout(): Observable<void> {
     return this.postRequest('logout').pipe(
       tap(() => this.setLoggedOut()),
-      catchError(error => throwError('Unable to log out.')),
+      catchError(() => throwError('Unable to log out.')),
     );
   }
 
   public isLoggedIn(): Observable<boolean> {
-    return of(!!(this._user && this._user.userid));
+    return of(!!(this.user && this.user.userid));
```

Comment:
this.loggedIn.next(Boolean(this.user && this.user.userid));
Choose one style. You can check style guide 22.6